script/spec-runner.js

```diff
@@ -1,23 +1,25 @@
 #!/usr/bin/env node
 
 const { ElectronVersions, Installer } = require('@electron/fiddle-core');
+
 const chalk = require('chalk');
+const { hashElement } = require('folder-hash');
+const args = require('minimist')(process.argv, {
+  string: ['runners', 'target', 'electronVersion'],
+  unknown: arg => unknownFlags.push(arg)
+});
```

Comment:
@MarshallOfSound, this is causing a build failure since `unknownFlags` isn't defined yet. Need to split out the `require('minimist')` like you did in some of the other files.

Reply:
Weird, the other ones I split out was because eslint couldn't autofix. Now I am suspicious that it might have autofixed something else into a broken state. Time to review every file one by one...